sunpy/database/database.py

```diff
@@ -384,10 +386,14 @@ def commit(self):
         """
         self.session.commit()
 
-    def _download_and_collect_entries(self, query_result, **kwargs):
+    def _download_and_collect_entries(self, query_result, **kwargs, path=''):
 
         client = kwargs.pop('client', None)
-        path = kwargs.pop('path', None)
+        if path is None:
+            path = kwargs.pop('path', None)
+        else:
+            path = path
```

Comment:
Is this `if,else` needed since path is now a string which is not `None` by default and a argument to the function instead of a kwarg

Reply:
I don't understand the reasoning for this, can you explain @jnanjekye